rails_values.gemspec

```
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name          = 'rails_values'
-  spec.version       = '1.2.0'
+  spec.version       = '1.2.1'
```

it's not a bug fix, so should be a minor version increase

```suggestion
  spec.version       = '1.3.0'
```